invenio_administration/menu/menu.py

```diff
@@ -62,6 +63,14 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_plus_menu_entries(self, current_menu, endpoint):
+        """Register plus menu items to a flask menu instance."""
+        current_menu.submenu("admin_plus.administration").register(
```

Comment:
this registers invenio-rdm-records specific menu in a generic invenio-administration module, which is not correct. For now it is enough if the menu is not registered via flask-menu but simply placed in the navbar with using `url_for("administration.oaipmh_create" ....`

Reply:
I would have done it that way if it was not because the endpoint of the administration is defined by a variable, if I'm not using flask menu, I think I should pass the endpoint variable to the templates of the administration, in order to be able to use the url_for properly (see https://github.com/inveniosoftware/invenio-administration/pull/72#discussion_r971833557).